worlds/dark_souls_3/docs/en_Dark Souls III.md

```diff
@@ -10,7 +10,10 @@ config file.
 In Dark Souls III, all unique items you can earn from a static corpse, a chest or the death of a Boss/NPC are randomized.
 This exclude the upgrade materials such as the titanite shards, the estus shards and the consumables which remain at 
 the same location. I also added an option available from the settings page to randomize the level of the generated 
-weapons( from +0 to +10/+5 )
+weapons( from +0 to +10/+5 )  
```

Comment:
```suggestion
weapons (from +0 to +10/+5)
```
forced newline should not be used before a new paragraph.
Pretty sure the suggestion is how spaces around `()` work in english. This may be different in different languages, so i didn't touch the french one.